Application/EdFi.Ods.Api/Helpers/AssemblyLoaderHelper.cs

```diff
@@ -142,35 +147,34 @@ public static IEnumerable<string> FindPluginAssemblies(string pluginFolder, bool
                 {
                     continue;
                 }
-
-                if (assemblyPath.Contains("Extensions"))
+                
+                string extensionFolder = Path.GetDirectoryName(assemblyPath);
+                if (Directory.Exists(extensionFolder))
                 {
-                    string extensionFolder = Path.GetDirectoryName(assemblyPath);
-                    var validationResult = validator.ValidateObject(extensionFolder);
-
-                    if (!validationResult.Any())
+                    string[] assemblyMetadataPath = Directory.GetFiles(
+                        extensionFolder, AssemblyMetadataSearchString, SearchOption.AllDirectories);
+                        
+                    if (assemblyMetadataPath.Length>0 && IsExtensionInAssemblyModelType(assemblyMetadataPath[0]))

```

Comment:
This looks to be based on an earlier version of https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/pull/247 for ODS-4664 and needs to be rebased on that after the appropriate changes are made there.